pages/Addons.md

```diff
@@ -56,7 +56,7 @@ DynaTech | ProfElements | <details><summary>expand</summary>An addon that adds r
 SlimyRepair | NCBPFluffyBear | <details><summary>expand</summary>An addon that allows players to repair Slimefun tools and weapons using customizable vanilla or Slimefun materials.</details> | [Readme](https://github.com/NCBPFluffyBear/SlimyRepair/blob/master/README.md) | [GitHub](https://github.com/NCBPFluffyBear/SlimyRepair) | [Builds](https://thebusybiscuit.github.io/builds/NCBPFluffyBear/SlimyRepair/master)
 SlimeCustomizer | NCBPFluffyBear | <details><summary>expand</summary>An addon that allows server owners to easily build their own Slimefun machines without any coding knowledge.</details> | [Readme](https://github.com/NCBPFluffyBear/SlimeCustomizer/blob/master/README.md) | [GitHub](https://github.com/NCBPFluffyBear/SlimeCustomizer) | [Builds](https://thebusybiscuit.github.io/builds/NCBPFluffyBear/SlimeCustomizer/master)
 GeneticChickengineering | kii-chan-reloaded | <details><summary>expand</summary>Genetic Chickengineering (GCE) is a mid- to late-game, resource-generating plugin heavily inspired by the Forge mod Chicken. GCE allows players to selectively breed special chickens to passively produce various vanilla resources, plus some Slimefun resources (mostly metal ore dusts).</details> | [Readme](https://github.com/kii-chan-reloaded/GeneticChickengineering) | [GitHub](https://github.com/kii-chan-reloaded/GeneticChickengineering) | [Releases](https://github.com/kii-chan-reloaded/GeneticChickengineering/releases)
-SimpleMaterialGenerators (SMG) | waleks647 | <details><summary>expand</summary>A simple addon with simple generators to reduce some grind.</details> | [Readme](https://github.com/waleks647/SMG/blob/master/README.md) | [GitHub](https://github.com/waleks647/SMG) | [Builds](https://thebusybiscuit.github.io/builds/waleks647/SMG/master)
+SimpleMaterialGenerators (SMG) | waleks647 | <details><summary>expand</summary>A simple addon with simple generators to reduce some grind.</details> | [Readme](https://github.com/Sefiraat/SMG/blob/master/README.md) | [GitHub](https://github.com/Sefiraat/SMG) | [Builds](https://thebusybiscuit.github.io/builds/Sefiraat/SMG/master)
```

Comment:
```suggestion
SimpleMaterialGenerators (SMG) | waleks647, Sefiraat | <details><summary>expand</summary>A simple addon with simple generators to reduce some grind.</details> | [Readme](https://github.com/Sefiraat/SMG/blob/master/README.md) | [GitHub](https://github.com/Sefiraat/SMG) | [Builds](https://thebusybiscuit.github.io/builds/Sefiraat/SMG/master)
```

Reply:
Currently I don't feel like I've done enough to be worthy of having my name there, Waleks did all the work making it. I may add this in should we add further gens in - this change is focused on making sure new players download the correct version really

Reply:
sure but seeing it does link to your github I would just add it for consistency sake
but if you feel strongly about it then its fine too